starter-code/views/celebrities/index.hbs

```diff
@@ -0,0 +1,6 @@
+<div>
+  <h2>
+    Celebrities
+  </h2>
```

Comment:
Aqui hay que recoger los datos que envias a la pagina:
{{#each celebrities}}
{{this}}
{{/this}}